app/code/Sergey/ServiceContract/Controller/Index/Index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Sergey\ServiceContract\Controller\Index;
+
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\View\Result\PageFactory;
+use Magento\Framework\App\Action\Action;
+
+/**
+ * Class Index
+ * @package Sergey\ServiceContract\Controller\Index\Index
+ */
+class Index extends Action
+{
+
+
+    /**
+     * Index resultPageFactory
+     * @var PageFactory
+     */
+    protected $resultPageFactory;
+
+    /**
+     * Index constructor.
+     * @param Context $context
+     * @param PageFactory $resultPageFactory
+     */
+    public function __construct(Context $context, PageFactory $resultPageFactory)
+    {
+        $this->resultPageFactory = $resultPageFactory;
+        return parent::__construct($context);
+    }
+
+    /**
+     * Function execute
+     * @return \Magento\Framework\View\Result\Page
+     */
+    public function execute()
+    {
+        return $this->resultPageFactory->create();
+    }
```

Comment:
please use `return $this->resultFactory->create(ResultFactory::TYPE_PAGE)` instead of current return code